imperfect-perfect.tex

```diff
@@ -433,6 +443,76 @@ \section{Interlude on perfectoid fields}
 Using Lemma~\ref{L:enough phi-invariants}, this follows as in the proof of \cite[Theorem~6.3.12]{kedlaya-liu1}.
 \end{proof}
 
+\begin{remark}
+Throughout this remark, assume that $A$ is a field.
+In this setting, the concept of $B$-pairs was introduced by Berger \cite{berger-b-pairs}
+in a purely algebraic fashion,
+without reference to the schemes defined in Definition~\ref{D:B-pairs}. Therein,
+the ring $\bB_{e,L,A}$ appears in connection with Fontaine's crystalline period ring
+$\bB_{\mathrm{crys}}$.
+
+The scheme $X_{L,A}$ introduced by Fargues--Fontaine is in some sense a ``complete curve'': in particular, it is a regular one-dimensional noetherian scheme.
+The space $P_{L,A,n}$ constitutes the sections of the $n$-th power of a certain ample line bundle on this scheme. This scheme admits something resembling an analytification in the category of adic spaces, in that there is a morphism into it from an adic space built out of the rings $\tilde{\calR}^{[s,r]}_{L,A}$, the pullback along which induces an equivalence of categories of coherent sheaves by analogy with Serre's GAGA theorem in complex algebraic geometry. This adic space in turn admits an infinite cyclic \'etale cover which is
+a ``quasi-Stein space'' whose global sections are the ring $\bigcap_{r>0} \tilde{\calR}^r_{L,A}$, on which the deck transformations act via the powers of $\varphi$.
+For more on this story, see the aforementioned references such as \cite{fargues-fontaine},
+and also \cite[\S 8.7, 8.8]{kedlaya-liu1} and \cite[\S 4.7]{kedlaya-liu2}.
+\end{remark}
+
+\begin{remark} \label{R:deeply ramified}
+Let $F/K$ be an algebraic extension. In the case where $F = \QQ_p(\mu_{p^\infty})$, we have seen already (Example~\ref{exa:cyclotomic}) that the completion of $F$ is a perfectoid field. 
+This property turns to be closely related to ramification of local fields; let us now recall the precise nature of this relationship.
+
+Coates--Greenberg \cite{coates-greenberg} define $F/K$ to be
+\emph{deeply ramified} if for every finite extension $F'$ of $F$, the trace map
+$\Trace: \frakm_{F'} \to \frakm_F$ is surjective. This holds in particular if $F/K$ is 
+\emph{arithmetically profinite} in the sense of Fontaine--Wintenberger
+\cite{fontaine-wintenberger} (see \cite[Corollary~1.5]{fesenko} for a detailed proof);
+the latter holds in turn if $F/K$ is a not totally unramified Galois extension whose Galois group is a $p$-adic Lie group, by a theorem of Sen \cite{sen-lie}.
```

Comment:
Yeah, I guess that example is perfectoid but not strictly APF because the ramification groups are not open. Will change this to "an infinite Galois extension with finite residue field".